Automattic/jetpack
# remove set version option of package

This PR moves forward with making the Connection package independent of the Jetpack plugin.

https://github.com/Automattic/jetpack/commit/3159d4c9913a857afe3a1fac84bfe3a5e8717de8

#### Changes proposed in this Pull Request:

* Moves a Jetpack specific connection routine to the plugin and out of the connection package

#### Jetpack product discussion


1199702491434901-as-1199702491434919

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






The call to this method at this point is there to solve an edge case scenario described in https://github.com/Automattic/jetpack/commit/3159d4c9913a857afe3a1fac84bfe3a5e8717de8. 

You can either try to reproduce the issue described in that commit, or simply add a debug to make sure the code is being triggered at the time we expect it to be triggered.
